src/legacy/server/saved_objects/export/objects_to_ndjson.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import stringify from 'json-stable-stringify';
+import { SavedObject } from '..';
+
+export function objectsToNdJson(savedObjects: SavedObject[]): string {
+  return savedObjects.map(doc => stringify(doc)).join('\n');
```

Comment:
Extracted from [route handler](https://github.com/elastic/kibana/blob/005170312d25535ae11cf696a5dc75f78992f486/src/legacy/server/saved_objects/routes/export.ts) for reuse